src/components/SidePanes/UsersPane.jsx

```diff
@@ -123,37 +140,62 @@ class UsersPane extends React.Component {
         {showEditors
           && (
           <div>
-            <div className="font-weight-bold">Course Editors</div>
+            <div className="font-weight-bold text-dark-700 mb-2">Course editors</div>
             {courseEditors.data.map(editor => (
               <User
                 key={editor.id}
                 userId={editor.id}
-                name={this.displayName(editor.user)}
+                name={editor.user.full_name}
+                email={editor.user.email}
                 onRemove={removeCourseEditor}
               />
             ))}
             {hasEditor
               || <div>All team members</div>}
             {showAddButton
               && (
-              <button type="button" className="btn btn-link p-0 usersPane-startAdd" onClick={this.startAddingUser}>
-                <Icon className="fa fa-plus" /> {hasEditor ? 'Add editor' : 'Set editor'}
+              <button type="button" className="btn btn-link text-dark-900 p-0 usersPane-startAdd" onClick={this.startAddingUser}>
+                <Add /> {hasEditor ? 'Add editor' : 'Set editor'}
               </button>
               )}
             {addingUser
               && (
               <div className="mt-2">
                 <hr />
-                <InputSelect
+                <div className="font-weight-bold text-dark-700 mb-2">Select an editor:</div>
+                <Select
                   name="add_editor"
-                  label={(
-                    <FieldLabel
-                      id="add_editor.label"
-                      text="Select an editor:"
-                    />
-                  )}
+                  placeholder="Search..."
+                  ref={this.selectRef}
+                  components={{
+                    IndicatorSeparator: () => null,
+                    Option,
+                  }}
+                  styles={{
+                    control: base => ({
```

Comment:
Add this to App.scss as classes

Reply:
classNames support was added to this component only very recently ([this month](https://react-select.com/styles))